python/metatomic_torch/metatomic/torch/ase_calculator.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         check_consistency=False,
         device=None,
         non_conservative=False,
+        do_energy_gradients_with_energy=True,
```

Comment:
I'm taking naming suggestions for this parameter!

Reply:
I kinda like `precompute_energy_gradients` but maybe `do_energy_gradients_with_energy` is the most descriptive..

Reply:
`do_gradients_with_energy` it is